src/components/ui/Chat/ChatBubble/ChatBubble.stories.tsx

```diff
@@ -1,37 +1,184 @@
 // Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import React from 'react';
+import React, {useState, ReactNode } from 'react';
 import { useTheme } from 'styled-components';
 
 import Flex from '../../Flex';
-import ChatBubble from '.';
+import ChatBubbleContainer, { Message } from './ChatBubbleContainer';
+import EditableChatBubble from './EditableChatBubble';
+import ChatBubble from './ChatBubble';
+import { PopOverItem, Modal, ModalHeader, ModalBody, ModalButtonGroup, ModalButton } from '../../../../';
+import { insertDateHeaders } from '../DateHeader/insertDateHeaders';
 import ChatBubbleDocs from './ChatBubble.mdx';
-import { PopOverItemProps } from '../../PopOver/PopOverItem';
 
-export default {
-  title: 'UI Components/Chat',
-  parameters: {
-    docs: {
-      page: ChatBubbleDocs.parameters.docs.page().props.children.type
-    }
-  },
-  component: ChatBubble
-};
+// export default {
+//   title: 'UI Components/Chat',
```

Comment:
Commented out! Also make sure Title is unique for each Chat Storie.

Reply:
I'm not sure about this one. We get a yellow console warning for this, I know, but this way storybook nicely organizes the stories into a folder structure that resembles the code's folder structure.